client/app/components/queries/ScheduleDialog.test.js

```diff
@@ -97,6 +98,57 @@ describe('ScheduleDialog', () => {
       });
     });
 
+    describe('TimeEditor', () => {
```

Comment:
This is awesome 👌 

But has one small issue: you assume 2 hours difference between the timezone and UTC. But what happens when DST kicks in (or the other way around)?

Reply:
I think it could be solved setting timezone for tests not using name, but explicitly `TZ=GMT-2:00` or something

https://github.com/getredash/redash/blob/549f878c98dcf515968c4f5b94b0fda59f50cc7c/package.json#L16

Reply:
That's a good save @arikfr :)
Now that I think of it, our other test snapshots might also fail when DST changes on March 29th! 😵

But I couldn't figure out an appropriate `TZ` value. Couldn't find docs, seems like it accepts only location strings and UTC. No offset. Plz help.

If no solution is found, might need to fallback to `TZ=UTC` and mock offset in test itself.

Reply:
@ranbena `TZ` is just a system variable and all programs should respect it:
![image](https://user-images.githubusercontent.com/12139186/53764499-279c7800-3ed6-11e9-8819-56e31feafb47.png)

Reply:
10x @kravets-levko but it just doesn't work.
I changed to a non-dst timezone instead, so we're good.
```
"test": "TZ=Africa/Khartoum jest",
```

Reply:
> but it just doesn't work.

Jest - :man_facepalming: